dms-service/src/main/java/de/muenchen/oss/swim/dms/adapter/out/dms/DmsAdapter.java

```diff
@@ -0,0 +1,82 @@
+package de.muenchen.oss.swim.dms.adapter.out.dms;
+
+import de.muenchen.refarch.integration.dms.api.IncomingsApi;
+import de.muenchen.refarch.integration.dms.api.ObjectAndImportToInboxApi;
+import de.muenchen.refarch.integration.dms.model.CreateIncomingAntwortDTO;
+import de.muenchen.refarch.integration.dms.model.CreateIncomingBasisAnfrageDTO;
+import de.muenchen.refarch.integration.dms.model.CreateObjectAndImportToInboxDTO;
+import de.muenchen.oss.swim.dms.application.port.out.DmsOutPort;
+import de.muenchen.oss.swim.dms.domain.exception.DmsException;
+import de.muenchen.oss.swim.dms.domain.model.DmsTarget;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.io.AbstractResource;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.client.WebClientResponseException;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class DmsAdapter implements DmsOutPort {
+    private final ObjectAndImportToInboxApi objectAndImportToInboxApi;
+    private final IncomingsApi incomingsApi;
+
+    private final static String DMS_APPLICATION = "SWIM";
+
+    @Override
+    public void putFileInInbox(final DmsTarget dmsTarget, final String fileName, final InputStream inputStream) {
+        log.debug("Putting file {} in inbox {}", fileName, dmsTarget);
+        final CreateObjectAndImportToInboxDTO request = new CreateObjectAndImportToInboxDTO();
+        request.setObjaddress(dmsTarget.coo());
+        try {
+            // FIXME directly use InputStream
+            final AbstractResource file = new NamedByteArrayRessource(fileName, inputStream.readAllBytes());
```

Comment:
`inputStream.readAllBytes()` sieht nicht danach aus, dass wir hier Streaming verwenden? Oder ist es das, was das FIXME oben meint? Kannst du das selbst fixen oder müssen sie dazu was in der Schnittstelle tun?

Reply:
Ne genau dafür ist das FIXME da, da muss man selbst irgendwie eine Klasse konstruieren und das hab ich nicht auf Anhieb hinbekommen.
Schau ich mir nochmal an.